apps/andi/lib/andi_web/live/search/manage_users_modal.ex

```diff
@@ -75,4 +85,20 @@ defmodule AndiWeb.Search.ManageUsersModal do
     |> Enum.map(fn org -> org.orgTitle end)
     |> Enum.join(", ")
   end
+
+  def selected_users(users, selected_users) do
+    Enum.map(selected_users, fn selected_user ->
+      case Enum.find(users, fn user -> user.id == selected_user end) do
+        nil -> Andi.Schemas.User.get_by_id(selected_user)
+        result -> result
```

Comment:
"if it's already in the search results no need to re-query", nice, very efficient.